releases/download/downloader.go

```diff
@@ -48,34 +48,20 @@ func (d *Downloader) Download(ctx context.Context, rawURL string, perm os.FileMo
 	}
 
 	target := d.targetPath(u)
-	tmp := target + ".tmp"
-
-	defer func() {
```

Comment:
defered functions are always run even in the event of a panic. No need to recover to execute behavior to then panic. Also renames are atomic so the only, cleanup necessary is to remove the tmp file.